### PR TITLE
Create ui-c8y-1020-0-9-missing-german-translation-in-successmessage-in-bulk-device-upload-6325-graft.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57157](https://cumulocity.atlassian.net/browse/MTM-57157)
Original PR: [6455](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6455)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-0-9-missing-german-translation-in-successmessage-in-bulk-device-upload-6325-graft.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Missing german translation in successmessage in bulk device upload (#6325) [GRAFT][release/cd] (#6455)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-0-9-missing-german-translation-in-successmessage-in-bulk-device-upload-6325-graft.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fixed missing German translation in success message for bulk device upload
```

File: content/change-logs/device-management/ui-c8y-1020-0-9-missing-german-translation-in-successmessage-in-bulk-device-upload-6325-graft.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57157
+version: 1020.0.9
+---
+Missing german translation in successmessage in bulk device upload (#6325) [GRAFT][release/cd] (#6455)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-0-9-missing-german-translation-in-successmessage-in-bulk-device-upload-6325-graft.md:17]: The Claude AI suggested to use the following body:
```suggestion
The success message displayed after uploading devices in bulk was not properly translated to German. This issue has been resolved and the success message now correctly appears in German for users with that language selected. The fix ensures a consistent user experience for German-speaking users when performing bulk device uploads.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@l2p- Please approve.

**Review by Oezge-Akin — APPROVED** [review]

**Pull request merged**